src/components/Dropdown/SubtleDropdown.tsx

```diff
@@ -4,6 +4,7 @@ import { CheckIcon } from "@heroicons/react/solid";
 import { Controller } from "react-hook-form";
 import clsx from "clsx";
 import { IItem } from "../../types/IItem";
+import { IncidentStatus } from "../../api/services/incident";
```

Comment:
SubtleDropdown is a generic component it shouldn't be referencing IncidentStatus - same applies for many of the other components - A generic component should not be aware of specific uses .

Reply:
> SubtleDropdown is a generic component it shouldn't be referencing IncidentStatus - same applies for many of the other components - A generic component should not be aware of specific uses . 

I'll fix that. Thanks for the review